elements/rh-secondary-nav/rh-secondary-nav-overlay.ts

```diff
@@ -0,0 +1,62 @@
+import { html, LitElement } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+
+import { ComposedEvent } from '@patternfly/pfe-core';
+import { pfelement, bound } from '@patternfly/pfe-core/decorators.js';
+
+import type { RhSecondaryNav } from './rh-secondary-nav.js';
+
+import styles from './rh-secondary-nav-overlay.css';
+
+export class SecondaryNavOverlayEvent extends ComposedEvent {
+  constructor(
+    public open: boolean,
+    public toggle: HTMLElement
+  ) {
+    super('overlay-change');
+  }
+}
+
+/**
+ * @summary A overlay to cover content when navigation is expanded.
+ */
+@customElement('rh-secondary-nav-overlay') @pfelement()
+export class RhSecondaryNavOverlay extends LitElement {
+  static readonly styles = [styles];
+
+  @state()
+  public open = false;
+
+  connectedCallback() {
+    super.connectedCallback();
+  }
+
+  render() {
```

Comment:
remove

Reply:
Fixed in 61bea434ccfe42654370827ed28825074a8c6f9d